js/main.js

```diff
@@ -1,17 +1,98 @@
-function getRandom(min, max) {
+const getRandom = (min, max) => {
   if (min>=max) {
     return 0;}
   else {
     return Math.floor(Math.random() * (max - min + 1)) + min;
   }
-}
-getRandom(1, 10);
+};
 
-function getRandomFraction(min, max, condition) {
+const getRandomFraction = (min, max, condition) => {
   if (min>=max) {
     return 0;}
   else {
     return  ((Math.random() * (max - min + 1)) + min).toFixed(condition);
   }
-}
-getRandomFraction (1, 10, 5);
+};
+
+const TYPES = [
+  'palace',
+  'flat',
+  'house',
+  'bungalow',
+  'hotel'
+];
+
+const TIMES = [
+  '12:00',
+  '13:00',
+  '14:00',
+];
+
+const FEATURES = [
+  'wifi',
+  'dishwasher',
+  'parking',
+  'washer',
+  'elevator',
+  'conditioner'
+];
+
+const PHOTO = [
+  'https://assets.htmlacademy.ru/content/intensive/javascript-1/keksobooking/duonguyen-8LrGtIxxa4w.jpg',
+  'https://assets.htmlacademy.ru/content/intensive/javascript-1/keksobooking/brandon-hoogenboom-SNxQGWxZQi0.jpg',
+  'https://assets.htmlacademy.ru/content/intensive/javascript-1/keksobooking/claire-rendall-b6kAwr1i0Iw.jpg'
+];
+
+const createUrl = () => {
+  const users=getRandom(1,10);
+  if (users===10) {
+    return `img/avatars/user${  users  }.png`;
+  } else {
+    return `img/avatars/user0${  users  }.png` ;
+  }
+};
+
+const getArray = (Array) => {
```

Comment:
```suggestion
const getRandomItems = (array) => {
```
с большой буквы в JS ничего просто так не называется)

название функции нужно давать какое-то более осмысленное